lib/immagine/image_processor_driver.rb

```diff
@@ -45,9 +44,10 @@ def process
       # CONVERT
       image_processor.convert_format!(conversion_format) if conversion_format
 
-      img         = image_processor.img
-      img_quality = quality
+      # QUALITY
+      img_quality = format_processor.quality.to_i || 85
```

Comment:
i'd maybe put the 85 in a constant at the top of the class, just so it's obvious